### PR TITLE
Improve README.md and rebar.config

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
I set out to improve some minor elements in `README.md` and ended up also tweaking `rebar.config`.

Don't merge this before #23 since I want to rebase on top of `master` once the former is closed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,63 @@
...
-=====
+===
 
-A rebar plugin
+![build](https://github.com/project-fifo/rebar3_lint/workflows/build/badge.svg)
 
-Build
------
-
-    $ rebar3 compile
```
Comment (paulo-ferraz-oliveira) [inline README.md:9]: I think this is general enough knowledge that doesn't require being included here.

File: README.md [hunk 2]
```diff
@@ -1,75 +1,63 @@
...
-Add the plugin to your rebar config:
+Add the plugin to your `rebar.config`:
 
 ```erlang
-{profiles,
-  [
-   {lint,  [{plugins, [{rebar3_lint, {git, "https://github.com/project-fifo/rebar3_lint.git", {tag, "0.1.11"}}}]}]}
-  ]
-}.
+{project_plugins, [
```
Comment (paulo-ferraz-oliveira) [inline README.md:15]: I selected a simpler approach to the configuration.

File: README.md [hunk 3]
```diff
@@ -1,75 +1,63 @@
...
-       filter => "rebar.config",
-       ruleset => rebar_config
-      }
-     #{dirs => ["."],
-       filter => "elvis.config",
-       ruleset => elvis_config
-      }
-    ]
-}
+[#{ dirs => ["apps/*/src/**", "src/**"],
```
Comment (paulo-ferraz-oliveira) [inline README.md:51]: I made this the same as the code.

File: README.md (hunk 1 above)
Comment (elbrujohalcon) [reply README.md:9]: I agree.

**Review by elbrujohalcon — APPROVED** [review]

**Comment by paulo-ferraz-oliveira** [comment]
Merging, since it's approved and rebased, as expected.

**Pull request merged**